GUIShop/shops.yml

```diff
@@ -338,6 +340,11 @@ FarmingDrops:
     type: SHOP
     id: SHULKER_SHELL
     buy-price: 2000.0
+  '36':
+    type: SHOP
+    id: WITHER_ROSE
+    buy-price: 100.0

```

Comment:
Maybe a little too cheap, you have to do a lot to get them.

Reply:
what about 150? if u make a farm u can get 2 double chests of wither rose in 1 hour